ibm_catalog.json

```diff
@@ -34,18 +34,18 @@
       ],
       "flavors": [
         {
-          "label": "Standard",
-          "name": "standard",
-          "install_type": "fullstack",
```

Comment:
Why was this removed @iamar7 ? I think this is causing an issue

Reply:
Added back in https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-governance/pull/35